test/units/testsuite-35.sh

```diff
@@ -0,0 +1,231 @@
+#!/usr/bin/env bash
+# SPDX-License-Identifier: LGPL-2.1-or-later
```

Comment:
@bluca @mbiebl Is it OK to use LGPL-2.1-or-later for the test script?

Reply:
@martinpitt ^

Reply:
@yuwata : I am fine with any copyleft license. The test currently is under LGPL-2.1+, see [debian/copyright](https://salsa.debian.org/systemd-team/systemd/-/blob/debian/master/debian/copyright#L139) file -- so this is even the status quo.

Thank you for integrating these into the upstream tests! :+1:

Reply:
Ah, there is a license file. Thanks.